README.md

```diff
@@ -78,6 +80,10 @@ for submitting pull requests to us.
 
 Visit [Contribute](https://github.com/PowerGridModel/power-grid-model/contribute) for a list of good first issues in this repo.
 
+## Citations
+
+Please cite using our latest [![DOI](https://zenodo.org/badge/8054429.svg)](https://zenodo.org/badge/latestdoi/8054429).
```

Comment:
Please paste the complete BibTex in here. Also please use the wording like

> If you are using Power Grid Model in your research work, please consider citing our library using the following references.

Reply:
Also add the same text in the introduction of the readthedocs

Reply:
will do 🙂

Reply:
In the BibTex block please add two references: the GitHub and the CIRED paper. I will provide you the CIRED paper BibTex later.

Reply:
i will have a look at how to do that and also try to add it to `CITATION.cff`

Reply:
@mgovers  You don't have to embed the CIRED paper in `CITATION.cff`. I am more talking about put the CIRED paper in BibTex in Readme and readthedocs.

Reply:
BibTex for CIRED paper

```bibtex
@inproceedings{Xiang2023,
author = {Xiang, Yu and Salemink, Peter and Stoeller, Bram and Bharambe, Nitish and van Westering, Werner},
booktitle = {CIRED 2023 - The 27th International Conference and Exhibition on Electricity Distribution},
title = {Power grid model: A high-performance distribution grid calculation library},
year = {2023},
volume={2023},
number = {},
pages={1-5}
}
```